cloud/new_ec2.go

```diff
@@ -128,6 +128,12 @@ func (m *ec2Manager) spawnOnDemandHost(h *host.Host, ec2Settings *NewEC2Provider
 		input.SecurityGroups = []*string{&ec2Settings.SecurityGroup}
 	}
 
+	grip.Debug(message.Fields{
```

Comment:
should we also add host provider name? (throughout?)

Reply:
done